modeling-cmds/src/def_enum.rs

```diff
@@ -266,6 +266,11 @@ define_modeling_cmd_enum! {
             pub entity_ids: Vec<Uuid>,
             /// The file format to export to.
             pub format: OutputFormat,
+            /// Chosen stem of the names of exported files.
```

Comment:
nitpick: let's try to avoid 'foo' in our public docs, that's a CS thing, and these docs will (hopefully) one day be read by MEs. I suggest "gear" or "clamp" or some kind of mechanical part name.